src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.managed.cs

```diff
@@ -482,7 +485,9 @@ void InitializeScrollViewer()
 					//	return OnScrollViewerViewChanged(pSender, pArgs);
 					//}));
 
+#if !__WASM__ // Workaround for https://github.com/unoplatform/uno/issues/14488
```

Comment:
This is the important part of the PR which works around the issue. The control still seems to work fine, so I'm not really sure what is the intent of hooking into ViewChanged. Do we expect people to grab the scroll viewer and try to programmatically scroll it, or is there other use cases for this? 😕 
Not really sure.

Reply:
@ajpinedam do you recall anything specific around this?

Reply:
This is coming from MUX code (the control is mostly ported version). Maybe there are some missing pieces or lifecycle differences though